lib/rate-limiter.js

```diff
@@ -98,12 +104,25 @@ class RateLimiter {
    *
    * @returns {Boolean}
    */
-  isEnabledFor (request) {
+  isEnabledFor(request) {
     const { enabled = true } = this.routeConfig(request)
 
     return enabled
   }
 
+  /**
+   * Whether to skip the the rate limiting or not
+   *
+   * @param {Request} request
+   *
+   * @returns {Boolean}
+   */
+  skip(request) {
+    const { skip = false } = this.routeConfig(request)
```

Comment:
Let's at first define `skip` is a plugin option, not as a route option. We can add `skip` as a route option in a later release.

Here, you should determine whether the user wants to skip the request or not by awaiting the `skip` function